### DISCUSSION TITLE
Which solution to support for chaining machine image builds?

### DISCUSSION THREAD

**Discussion by sestegra** [vmware/packer-examples-for-vsphere#229] [Ideas]
In order to enable chaining of multiple layered images, the `vmware-clone` builder needs to be used.

Which alternative to using, full clone or linked clone?
If the linked clone option is selected, the `create_snapshot` parameter should be added to the current project.

What do you think about it?

See reference
https://www.packer.io/guides/packer-on-cicd/pipelineing-builds

![image](https://user-images.githubusercontent.com/109060/180646541-8d88dbec-d7aa-4ecf-bf39-0b1e9e549404.png)

**Comment by tenthirtyam** [comment]
I have more of a preference for full clones for a server OS and linked clones for a desktop OS (e.g., Windows 11/10 for Horizon).

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.
